addons/mail/__manifest__.py

```diff
@@ -175,6 +175,7 @@
         ],
         "web.assets_frontend": [
             "mail/static/src/utils/common/format.js",
+            'mail/static/src/utils/common/pdf_thumbnail.js',
```

Comment:
Would put in folder `frontend/` and have `frontend/**/*` in manifest.

Reply:
I don't understand. The frontend folder doesn't exist? I don't see why I would create a whole new folder for this when we already have utils?

Reply:
Oops sorry comment was very badly worded, I meant that the file shouldn't be added manually in manifest and instead should use `folder/**/*` logic.

Things is that in theory common should have it available in frontend but it's not because frontend needs minimal footprint, so normally should put files in the right folder.

We can look at it later, this is more a concern for how to move the other utils in another folder and/or we have to invent a blacklist naming convention in our folders.

Reply:
Why do we need `pdf_thumbnail` in front-end exactly?

Usually front-end stuff should be lazy-loaded either through the livechat embed bundle or the chatter bundle.